### ISSUE TITLE
Transciption of names with -yos

### ISSUE THREAD

**Issue by MagdaKrzyz** [BetaMasaheft/Dillmann#286]
Dear All, 
I have noticed that there are two ways of transliterating names with -yos at the end. In some of them there is shwa before the -yos and in some not. Examples:
ቀላምዮስ qalāmyos
vs
ከላውድዮስ kalāwdǝyos

Now it is pronounced without the shwa. I tend to this solution. What do you think?
Magda

**Comment by abausi** [comment]
I agree. Let's do so and let's gradually normalize everything to -yos.

**Comment by thea-m** [comment]
Sorry to intrude (it seems to be my intruding day), I just read this accidentally and wondered whether it is relevant for BM to, in which case I will add the proposed transliteration to https://betamasaheft.eu/Guidelines/?id=dubious-spelling.
What about names in which this transliteration would lead to three consonant clusters? ከላውድዮስ, in BM I also found አውስግንዮስ and ልብድዮስ (for which the traditional pronunciation seems to be Lǝbdǝyos)?
